CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.4.1] - 2022-09-29
+### Added
+- Expose new connection pool configuration options (`connectionTtlMs`, `idleConnectionValidationMs`, `evictIdleConnectionsAfterSecs`) 
+- Document configuration builder API
```

Comment:
we should probably add the new config options here: https://experienceleague.adobe.com/docs/target-dev/developer/server-side/java/initialize-sdk.html?lang=en

Reply:
Right, we should update the docs after releasing the new SDK version

Reply:
Opened doc update PR and asked Mark to review & merge it - https://git.corp.adobe.com/AdobeDocs/target-dev.en/pull/202